frontend/src/createStream/StreamingTable.jsx

```diff
@@ -45,14 +44,11 @@ export default class StreamingTable extends React.Component {
   }
 
   handleSubmit(event) {
+    event.preventDefault();
     const { selectedReceiverID, selectedSenderID } = this.state;
     if (selectedReceiverID !== "" && selectedSenderID !== "") {
-      axios.post(process.env.REACT_APP_STREAM, {
-        inputChannelId: selectedReceiverID,
-        outputChannelId: selectedSenderID
-      });
+      // createStream( selectedReceiverID, selectedSenderID);
```

Comment:
Leave in the `axios.post` call so that the feature is never broken on master, and we'll change to using `createStream` when we merge into auth. (add a TODO so I don't forget when i'm rebasing)

Reply:
handled in 82cb59a